Localize/lang/strings.json

```
@@ -1259,6 +1264,7 @@
   "_LvLksz.comment": "Loading outputs text",
   "_LvpxiA.comment": "Column header for connection gateway",
   "_Lwe7tQ.comment": "Timezone value ",
+  "_LxPh9u.comment": "accessability label for a button to copy all text in a value box",
```

First letter should be capitalized:

```suggestion
  "_LxPh9u.comment": "Accessibility label for a button to copy all text in a value box",
```